frontend/src/metabase/query_builder/components/notebook/types.ts

```diff
@@ -52,6 +52,7 @@ export interface NotebookStepAction {
 export interface NotebookStepUiComponentProps {
   step: NotebookStep;
   topLevelQuery: Query;
+  stageIndex: number;
```

Comment:
Can't we just use `step.stageIndex`? It seems that they're always the same.

It feels weird to have 2 sources of this information.

Reply:
what do you think about unification like this https://github.com/metabase/metabase/pull/36267 ?